final_task/rss_reader/rss_reader.py

```diff
@@ -0,0 +1,178 @@
+import logging
+import os
+import sys
+from typing import List
+
+directory_to_module = os.path.abspath(os.path.dirname(__file__))
+sys.path.append(directory_to_module)
+
+import single_article
+import html_converter
+import pdf_converter
+import articles_handler
+import argparse_handler
+import custom_error
+
+LOG_FILE_NAME = os.path.join(directory_to_module, 'app.log')
+
+
+def start_logging() -> None:
```

Comment:
I would recommend to rename this function, since you actually creating and configuring logger.